.github/workflows/test.yml

```diff
@@ -82,11 +82,12 @@ jobs:
           WORKSPACE: ${{ github.workspace }}
           PARAMETERS: ${{ steps.prepare.outputs.parameters }}
         run: |
+          chmod -R 777 "${WORKSPACE}/${OUTPUT_DIR}"
```

Comment:
🤣 777 is maybe overkill

Reply:
https://github.com/RMI-PACTA/workflow.pacta/issues/65